contrib/pg_tde/documentation/CONTRIBUTING.md

```diff
@@ -1,10 +1,90 @@
+# Contributing guide
+
+Welcome to `pg_tde` - the Transparent Data Encryption extension for PostgreSQL!
+
+We're glad that you would like to become a community member and contribute to this project.
+
+You can contribute in one of the following ways:
+
+1. Reach us on our [Forums](https://forums.percona.com/c/postgresql/25).
+2. Submit a bug report or a feature request
+3. Submit a pull request (PR) with the code patch
+4. Contribute to documentation
+
+## Prerequisites
+
+Before submitting code contributions, we ask you to complete the following prerequisites.
+
+### 1. Sign the CLA
```

Comment:
@ImTheKai do we want to configure CLA assistant for this repo?

Reply:
Yes, we have to actually. Who can take care of that?

Reply:
Setup seems easy, but not sure what agreement are we using and we have to specify that, maybe IT?

Reply:
Reached out to IT, will solve this, once I have feedback.